src/js/decafmud.js

```diff
@@ -1872,7 +1940,7 @@ DecafMUD.options = {
 		handlecolor	: true,
 		fgclass		: 'c',
 		bgclass		: 'b',
-        fntclass    : 'fnt',
+		fntclass    : 'fnt',
```

Comment:
This change doesn't seem to be relevant to this PR.

Reply:
https://github.com/Tiger-Tom/DecafMUD/commit/aa44499e6c0fc0d9a8b96a250f20dc26930bee3d
I think that this was just a result of the flattening of leading spaces to tabs.